### ISSUE TITLE
.NET 8 upgrade

### ISSUE THREAD

**Issue by DavidWiseman** [trimble-oss/dba-dash#779]
.NET 8 is the latest LTS version with support until 2026.   .NET 6 support ends Nov 2024 - still a bit of time left but it's worth upgrading early.   Upgrading to .NET 8 will bring some free performance improvements.  Some packages used by DBA Dash have recently had issues upgrading to the latest version which seems to be caused by the referenced packages being upgraded.  
The .NET 8 upgrade should also fix the issue where the application doesn't start if users have an older version of .NET 6 installed (before 6.0.2)

Considerations:
* Users will need to install .NET 8.  This makes upgrading to the latest version of DBA Dash slightly more difficult.  The upgrade script could be changed to check for .NET 8 and prevent the upgrade until this is installed.
* The upgrade to .NET 8 should be called out in the release blog.
* Compatibility with .NET 8 isn't known at this point.

**Comment by DavidWiseman** [comment]
Export to Excel is broken by DocumentFormat.OpenXml update from 2.20.0 to 3.0.0

`SLDocument sl = new();`
```
System.InvalidProgramException
  HResult=0x8013153A
  Message=Common Language Runtime detected an invalid program.
  Source=SpreadsheetLight
  StackTrace:
   at SpreadsheetLight.SLDocument.LoadStylesheet()
   at SpreadsheetLight.SLDocument.InitialiseStylesheetWhatNots(SLThemeTypeValues ThemeType)
   at SpreadsheetLight.SLDocument..ctor()
   at DBADashGUI.Common.SaveDataGridViewToXLSX(DataGridView& dgv, String path) in C:\Git\TrimbleOSS\dba-dash\DBADashGUI\Common.cs:line 162
   at DBADashGUI.Common.PromptSaveDataGridView(DataGridView dgv) in C:\Git\TrimbleOSS\dba-dash\DBADashGUI\Common.cs:line 155
   at DBADashGUI.Common.PromptSaveDataGridView(DataGridView& dgv) in C:\Git\TrimbleOSS\dba-dash\DBADashGUI\Common.cs:line 136
   at DBADashGUI.Summary.ExportSummaryToolStripMenuItem_Click(Object sender, EventArgs e) in C:\Git\TrimbleOSS\dba-dash\DBADashGUI\Checks\Summary.cs:line 695
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(HWND hWnd, MessageId msg, WPARAM wparam, LPARAM lparam)
```

Rolling back to 2.20.0

**Comment by DavidWiseman** [comment]
This is included in version 3.0.   🚀
